sbnanaobj/StandardRecord/SRCaloPoint.h

```diff
@@ -20,6 +20,8 @@ namespace caf
       float dedx; //!< dQ/dx [ADC/cm] -- pre calibration and electron lifetime correction
       float pitch; //!< Track pitch [cm]
       float t; //!< Time of deposition [ticks]
+      float efield; //! |E| [kV/cm]
+      float phi; //! angle between the E-field and track dir for the hit, used in the GnocchiCalorimetry module
```

Comment:
Please use the marker `//!<` instead of `//!` (the latter will stick the comment to the _next_ line).
```suggestion
      float efield; //!< |E| [kV/cm]
      float phi; //!< angle between the E-field and track dir for the hit, used in the GnocchiCalorimetry module
```

Reply:
Hi Gianluca, thank you for the review! It is updated!